data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,420 @@ mission "Remnant: Face to Maw 2B"
+			`	Passing through the cavernous opening into the great spaceport dome, there is a shimmer in the air next to you and a starling fades into view, settling onto the platform beside you. The pilot signs a quick "Greetings, anticipating talking with you." before heading back toward their lock. Behind them, the Starling continues its shutdown checks without any supervision at all.`
+			`	As you finish your own shutdown routine, your commlink informs you that your account has been updated with <payment>.`
+
+
+
+ship "Starling" "Starling (Keystone)"
+		sprite "ship/starling"
+		thumbnail "thumbnail/starling"
+		attributes
+			category "Light Warship"
+			licenses
+				Remnant
+			cost 8800000
+			"shields" 8800
+			"hull" 4900
+			"required crew" 6
+			"bunks" 15
+			"mass" 270
+			"drag" 3.8
+			"heat dissipation" 0.7
+			"fuel capacity" 900
+			"ramscoop" 1
+			"cargo space" 36
+			"outfit space" 312
+			"weapon capacity" 83
+			"engine capacity" 94
+			"shield generation" 1.8
+			"shield energy" 1.2
+			"hull repair rate" 0.8
+			"hull energy" 0.6
+			"cloak" .02
+			"cloaking energy" 5
+			"cloaking fuel" .1
+			"outfit scan power" 12
+			"outfit scan speed" 1
+			"tactical scan power" 26
+			weapon
+				"blast radius" 80
+				"shield damage" 800
+				"hull damage" 400
+				"hit force" 1200
+		outfits
+			"Inhibitor Cannon" 3
+			"Point Defense Turret"
+			
+			"Epoch Cell"
+			"Crystal Capacitor"
+			"Emergency Ramscoop" 2
+			"Quantum Keystone"
+			"Salvage Scanner"
+			
+			"Forge-Class Thruster"
+			"Forge-Class Steering"
+			"Crucible-Class Steering"
+			"Hyperdrive"
+
+		engine -16 82 0.7
+		engine 16 82 0.7
+		engine 0 90 1
+		gun 0 -99 "Inhibitor Cannon"
+		gun -12 -72 "Inhibitor Cannon"
+		gun 12 -72 "Inhibitor Cannon"
+		gun -18 -49
+		gun 18 -49
+		turret 0 2 "Point Defense Turret"
+		leak "remnant leak" 30 4
+		leak "remnant leak sparkle" 30 10
+		explode "tiny explosion" 40
+		explode "small explosion" 20
+		explode "medium explosion" 10
+		"final explode" "final explosion small" 1
+		description "One of the strangest and most useful things that the Remnant have discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They have designed their scout ships to cloak when needed in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."
+		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."
+
+
+
+mission "Remnant: Crystal Research 5"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Keystone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
```

Comment:
I gave it those personalities so that it would act effectively like other Remnant - except that it would run away if it got too badly damaged. This is not the same as uninterested, which is fairly bland and basically just translates "ignores the player."